src/templates/src/main.ts

```diff
@@ -1 +1,12 @@
-/* Put your code here */
+import createProjector from 'dojo-widgets/createProjector';
+import d from 'dojo-widgets/d';
+import createHelloWorld from './widgets/createHelloWorld';
+
+const root = document.getElementsByTagName('my-app')[0];
```

Comment:
`document.querySelector('my-app')` is slightly cleaner (as you don't need the array access)